src/app/src/index.js

```diff
@@ -8,22 +8,32 @@ import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 import * as actions from './actions';
-import createStoreWithMiddleware from './store';
-import reducers from './reducers';
+import configureStore from './store';
 
-const store = createStoreWithMiddleware(reducers);
+const store = configureStore();
 assignAll(actions, store);
 
-render(
-    <Provider store={store}>
-        <BrowserRouter>
-            <div>
-                <Route exact path='/' component={App} />
-            </div>
-        </BrowserRouter>
-    </Provider>,
-    document.getElementById('root')
-);
+const render = Component => {
+    return ReactDOM.render(
```

Comment:
I think this can just be `render` and the import update can be removed.

Reply:
Nevermind -- I misread this. Made a note above about using a different name for the wrapped function.